docs/source/support/contribution.rst

```diff
@@ -387,4 +353,4 @@ For example, for a new ``patch`` release, we would do:
   (nucypher)$ make release bump=patch
 
 3. The previous step triggers the publication webhooks on CircleCI.
```

Comment:
Do we still use CircleCI for release publication?

Reply:
No... this appears to be some leftover reference to circleCI.  Let's make an update here alongside a package publication GH action instead of changing it in this PR.